NetKAN/TextureReplacerReplaced.netkan

```diff
@@ -0,0 +1,17 @@
+{
+	"spec_version": "v1.4",
+	"name": "TextureReplacerReplaced",
+	"$kref": "#/ckan/github/HaArLiNsH/TextureReplacerReplaced",
+	"identifier": "TextureReplacerReplaced",
+	"release_status": "testing",
+	"ksp_version": "1.3.0",
+	"author": ["shaw", "RangeMachine", "HaArLiNsH"],
+	"abstract": "Kerbal personalisation, IVA suits on Kerbin and texture replacement and improvements.",
+	"resources": {
+		"homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/161898",
+		"repository": "https://github.com/HaArLiNsH/TextureReplacerReplaced"
+	},
+	"provides": ["TextureReplacer"],
```

Comment:
Also add `"conflicts": [ { "name": "TextureReplacer" } ],` so people cant install both mods, just to be sure.

Reply:
done